coffee_roaster_market/apps/account/permissions.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import permissions
+
+
+class CurrentUserOrAdmin(permissions.IsAuthenticated):
+    def has_object_permission(self, request, _, obj):
```

Comment:
Same as: https://github.com/AleksanderPawlak/coffee-roaster-market/pull/12#discussion_r654963786

Reply:
Add in entire file.